### ISSUE TITLE
UrlShotner: bridg.li inactive by pyfunceble

### ISSUE THREAD

**Issue by spirillen** [ShadowWhisperer/BlockLists#32]
Do you know anything else about this domain? as there is a DNS assigned but no IP

**Comment by spirillen** [comment]
```
drill bridg.li -T
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
li.     172800  IN      NS      a.nic.li.
li.     172800  IN      NS      b.nic.li.
li.     172800  IN      NS      d.nic.li.
li.     172800  IN      NS      e.nic.li.
li.     172800  IN      NS      f.nic.li.
bridg.li.       3600    IN      NS      ns2.globalinternet.pro.
bridg.li.       3600    IN      NS      ns1.globalinternet.pro.
```

**Comment by ShadowWhisperer** [comment]
I do not. That is weird :-/

Whois.com - Show domain name is invalid
`https://www.whois.com/whois/bridg.li`

IBM Cloud has no info on it.
`https://exchange.xforce.ibmcloud.com/url/bridg.li`

Google must have crawled it at one point.
`URL shortener: Great! Your domain name is working!
URL shortener, QR codes, splash pages, tracking, teams, geotargeting, device targeting and more.`

I've noticed a ton of domains that say they are a URL shortener, but have not seen real world use. It's kind of why I stopped updating the main list. They show up for a few days or weeks, then they are gone. Everything in the list below is from automated results.
`https://raw.githubusercontent.com/ShadowWhisperer/BlockLists/master/Testing/UrlShortener`
